addons/point_of_sale/static/src/xml/Popups/CashMovePopup.xml

```diff
@@ -2,19 +2,19 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="point_of_sale.CashMovePopup" owl="1">
-            <Draggable>
-                <div class="popup drag-handle">
+        <div role="dialog" class="modal-dialog">
```

Comment:
May I know why you still wrap this component with this particular `div`? I don't think it's needed if you are using this popup via `showPopup` method. The place where this popup will be mounted is already wrapped in this `div`. Check the template of `PosPopupController`.

Reply:
Thanks for catching that! I will remove it!